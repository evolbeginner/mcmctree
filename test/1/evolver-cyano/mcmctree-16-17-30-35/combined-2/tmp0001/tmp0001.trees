  1

(Xenococcus_sp_PCC_7305, Moorea_producens_3L, ((Oscillatoria_sp_PCC_10802, Crinalium_epipsammum_PCC_9333), Nostoc_sp_PCC_7107));
