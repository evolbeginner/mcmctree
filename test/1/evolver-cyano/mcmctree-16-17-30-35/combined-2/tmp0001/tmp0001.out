
Printing out site pattern counts


         5         21  P

Xenococcus_sp_PCC_7305          AAACCCCCGG GTTTTTTTTT T
Moorea_producens_3L             AATCCTTTAA GCCCTTTTTT T
Oscillatoria_sp_PCC_10802       AACCCCCTAG GCCTACCCTT T
Crinalium_epipsammum_PCC_9333   AATCTCTTAG GCCTACCTCT T
Nostoc_sp_PCC_7107              AGCCCCTTAG GCTTACTTTC T



   24    3    1   33    1    2    1    3    2    1    7    2    1    1    1
    1    1    1    1    1   12


BASEML (in paml version 4.9j, October 2019)  tmp0001.txt  REV dGamma (ncatG=5)
Frequencies..
                                    T      C      A      G
Xenococcus_sp_PCC_7305         0.22000 0.40000 0.28000 0.10000 GC = 0.500
Moorea_producens_3L            0.25000 0.38000 0.30000 0.07000 GC = 0.450
Oscillatoria_sp_PCC_10802      0.18000 0.44000 0.30000 0.08000 GC = 0.520
Crinalium_epipsammum_PCC_9333  0.21000 0.41000 0.30000 0.08000 GC = 0.490
Nostoc_sp_PCC_7107             0.21000 0.41000 0.27000 0.11000 GC = 0.520

Homogeneity statistic: X2 = 0.03141 G = 0.03127 

Average                        0.21400 0.40800 0.29000 0.08800

# constant sites:     76 (76.00%)
ln Lmax (unconstrained) = -214.668169

Distances: TN93 (kappa)  (alpha set at 0.50)
This matrix is not used in later m.l. analysis.

Xenococcus_sp_PCC_7305  
Moorea_producens_3L    0.2540(45.8783)
Oscillatoria_sp_PCC_10802    0.2278(17.5852)  0.1599(23.7085)
Crinalium_epipsammum_PCC_9333    0.3041(22.0045)  0.1320(19.7018)  0.0686(999.0000)
Nostoc_sp_PCC_7107    0.2660(28.6063)  0.1918(45.3924)  0.1116(999.0000)  0.1301(999.0000)

TREE #  1:  (1, 2, ((3, 4), 5));  MP score: 28.00
lnL(ntime:  7  np: 13):   -239.170351    +0.000000
   6..1     6..2     6..7     7..8     8..3     8..4     7..5  
 0.125385 0.043747 0.028115 0.009622 0.032861 0.024640 0.066988 1.017047 0.116018 0.000010 0.000010 0.000010 1.476043
SEs for parameters:
 0.058647 0.031554 0.039425 0.015587 0.025132 0.020307 0.037532 0.641329 0.190332 2.077515 0.305958 1.251852 2.885043

tree length =   0.33136

(Xenococcus_sp_PCC_7305, Moorea_producens_3L, ((Oscillatoria_sp_PCC_10802, Crinalium_epipsammum_PCC_9333), Nostoc_sp_PCC_7107));

(Xenococcus_sp_PCC_7305: 0.125385, Moorea_producens_3L: 0.043747, ((Oscillatoria_sp_PCC_10802: 0.032861, Crinalium_epipsammum_PCC_9333: 0.024640): 0.009622, Nostoc_sp_PCC_7107: 0.066988): 0.028115);

Detailed output identifying parameters

Parameters  in the rate matrix (REV) (Yang 1994 J Mol Evol 39:105-111):


Rate parameters:    1.01705  0.11602  0.00001  0.00001  0.00001
Base frequencies:   0.21400  0.40800  0.29000  0.08800
Rate matrix Q, Average Ts/Tv =  15.8738
   -1.845758    1.707322    0.138433    0.000004
    0.895507   -0.895523    0.000012    0.000004
    0.102154    0.000017   -0.464244    0.362074
    0.000009    0.000017    1.193198   -1.193223

alpha (gamma, K=5) =  1.47604
rate:   0.18572  0.47172  0.79011  1.23562  2.31684
freq:   0.20000  0.20000  0.20000  0.20000  0.20000

